scholia/app/templates/pathway.html

```diff
@@ -120,6 +121,9 @@
   $.getJSON(sparqlURL, function(response) {
     const simpleData = sparqlDataToSimpleData(response);
     const dataValues = simpleData.data[0];
+    if ("description" in dataValues) {
+      $("#description").text(dataValues.description);
+    }
     if ("organism" in dataValues) {
       $("#Organism").after('<a href="'+dataValues.organism+'">'+dataValues.organismLabel+'</a>'); 
```

Comment:
Do we need an escape of dataValues.organismLabel?

Sorry for the delay in my response.

Reply:
I probably just need coffee... but I'm not sure what you mean by escape and why we would need one here?  And you say, "dataValues.organismLabel", but I'm assuming you mean, "dataValues.description."

Do you mean, we need an "else" case? The "else" is to do nothing, i.e., not to add any text to the `description` div. It will simply be blank if no description is available (like it is currently).

Reply:
I mean in the unlikely case that dataValues.organismLabel contains, e.g., "<script..." the the "<" should be escaped. I suppose "dataValues.organism" cannot be other than Q\d+ so that is not a problem. Possibly Q4115189 can be used.

Reply:
Ok. Happy to do that. I've opened an issue to do this in another PR, however, since this one is just about adding the description handling code. The line you want changed is not part of this PR.

https://github.com/wikipathways/scholia/issues/4

Reply:
@fnielsen Looks like I should add the escaping of "<" to this utils function and then use it for dataValues.organismLabel: https://github.com/fnielsen/scholia/blob/master/scholia/utils.py#L7. Right?

And I can use sanitize_q on dataValues.organism.

Again, I'll do this in a separate PR. Anything else for this PR on description handling?

Reply:
I wonder if `.text(dataValues.organismLabel)` can be used? This is already done above. I suppose the Q-identifier would need no sanitization.

Reply:
This PR is a few lines about adding descriptions to the template. Your questions appear to be about how the organism info could be improved. That code was merged in a prior PR and is already live. 

I've started a new issue (https://github.com/wikipathways/scholia/issues/4) to work on those suggestions, but don't want to prepare the next PR until this one is clear. 

Any other questions or suggestions for this PR?